tck/README.md

```diff
@@ -268,8 +274,8 @@ public class MySubscriberTest extends SubscriberBlackboxVerification<Integer> {
 ```
 
 
-The `createElement` method MAY be called from multiple
-threads, so in case of more complicated implementations, please be aware of this fact.
+The `createElement` method MAY be called from multiple threads, so in case of more complicated implementations, 
```

Comment:
method MAY be called **concurrently** from multiple threads, so in case of more complicated implementations,